test/ui-test/contentAssistTest.ts

```diff
@@ -38,8 +40,17 @@ export function contentAssistSuggestionTest(): void {
     });
 
     afterEach(async function () {
-      const { exec } = await require('child_process');
-      exec('rm ~/kustomization.yaml');
+      /* eslint-disable */
+      const fs = require('fs');
+      const homeDir = os.homedir();
+      const pathtofile = path.join(homeDir, 'kustomization.yaml');
+
+      if (fs.existsSync(pathtofile)) {
+        console.log(`yaml file does exist - removing`);
```

Comment:
I guess there is no need  for console output, right?

Reply:
Fixed